pkgs/by-name/an/angular-language-server-bin/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  nodejs,
+  makeBinaryWrapper,
+  runCommand,
+  angular-language-server-bin,
+  writeShellApplication,
+  curl,
+  common-updater-scripts,
+  jq,
+  unzip,
+}:
+
+let
+  owner = "angular";
```

Comment:
Just hardcore those. They will likely never ever change.

Reply:
You are right that they probably never change but I think it does not hurt either so I keep them in if that is ok.